ricky9667/HushKeyboard
# Create introduction page

## Description

This PR creates an introduction page for users to setup keyboard easier.

## How to verify

1. Setup page should show each section after the last button is tapped.
2. Textfield in the introduction page should verify if user types `R U R' U', and shows dialog if true.

## Screenshots /  Videos

https://github.com/user-attachments/assets/27027644-7c53-4537-bfb1-cac232dcd0df
